BedBrigade/Client/Pages/Admin/Login.razor.cs

```diff
@@ -0,0 +1,161 @@
+        [Inject] private AuthenticationStateProvider AuthenticationStateProvider { get; set; }
+
+        [Parameter] public string? User { get; set; }
+        [Parameter] public string? Password { get; set; }
+
+        protected UserLogin loginModel = new();
+        protected string DisplayError { get; set; } = "none;";
+        protected string Error = "";
+        protected StringValues returnUrl;
+
+        protected SfToast? ToastObj;
+        protected string? ToastContent;
+        protected int ToastTimeout;
+        protected string ToastTitle = string.Empty;
+        protected string? errorMessage;
+        protected InputText? inputTextFocus;
+
+        public bool showPassword { get; set; }
+        public string? passwordType { get; set; }
+        public string Message { get; set; }
+
+        protected override void OnInitialized()
+        {
+            loginModel.Email = User;
+            loginModel.Password= Password;
+            Console.WriteLine("In Login Razor Initialize");
+            passwordType = "password";
+
+            var uri = NavigationManager.ToAbsoluteUri(NavigationManager.Uri);
+            if (QueryHelpers.ParseQuery(uri.Query).TryGetValue("returnUrl", out var url))
+            {
+                returnUrl = url;
+            }
+
+        }
+
+        protected async Task HandlePassword()
+        {
+            if (showPassword)
+            {
+                passwordType = "password";
+                showPassword = false;
+            }
+            else
+            {
+                passwordType = "text";
+                showPassword = true;
+            }
+        }
+
+        protected async Task HandleLogin()
+        {
+            var result = await AuthService.Login(loginModel);
+            if (result.Success)
+            {
+
+                await _local.SetItemAsync("authToken", result.Data);
+                await AuthenticationStateProvider.GetAuthenticationStateAsync();
+
+                NavigationManager.NavigateTo(returnUrl);
+            }
+            else
+            {
+                try
+                {
+                    
+                    ToastTitle = "Login Error";
+                    ToastContent = "The credentials entered are not valid.<br/> Please try again.";
+                    ToastTimeout = 5000;
+
+                    await ToastObj.ShowAsync();
+
+                    errorMessage = result.Message;
+                    DisplayError = "block;";
+                    loginModel.Email = string.Empty;
+                    loginModel.Password = string.Empty;
+                    if (inputTextFocus.Element != null)
+                    {
+                        await inputTextFocus.Element.Value.FocusAsync();
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine(ex.ToString());
+                }
+;
+            }
+        }
+
+
+        //protected async Task HandleLogin()
```

Comment:
Are we okay to delete the commented code?

Reply:
Yes.